recipe/meta.yaml

```diff
@@ -1,13 +1,13 @@
 {% set name = "optuna" %}
-{% set version = "3.0.5" %}
+{% set version = "3.1.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 9e39869e76c8b57b7b0f9badb31853fffed29e1da52b508fd78bd1d5307b79c8
+  sha256: 96c7c92860c8692d3aa569d749e72b121422cb4af0ed3ad4bfbc445b61416919
```

Comment:
The hash value seems to be correct.
https://pypi.org/project/optuna/#copy-hash-modal-0b65cdad-22d6-4329-8323-4af10140027a